glide.yaml

```diff
@@ -1,3 +1,10 @@
-package: github.com/deis/pkg/time
+package: github.com/deis/controller-sdk-go
 import:
-- package: github.com/deis/pkg/time
+- package: github.com/deis/pkg
+  subpackages:
+  - time
+- package: github.com/goware/urlx
+- package: github.com/PuerkitoBio/purell
```

Comment:
`glide get github.com/goware/urlx` didn't pull them in. Is that the default behavior or do I need to pass a flag?